MediaGalleryUi/view/adminhtml/web/js/image-uploader.js

```diff
@@ -115,18 +115,19 @@ define([
          */
         showErrorMessage: function (data) {
             data.files.each(function (file) {
-                this.mediaGridMessages().add('error', $.mage.__('Cannot upload <b>' + file.name + '</b>. This file is not supported'));
+                this.mediaGridMessages().add(
+                    'error',
+                    $.mage.__('Cannot upload <b>' + file.name + '</b>. This file is not supported')
```

Comment:
Let's change the `This file is not supported` to `The file format is not supported` :)

Reply:
good catch :)